### ISSUE TITLE
file input element inside a-router not working

### ISSUE THREAD

**Issue by ranglang** [atomicojs/hooks#2]
```
const result1 = demoResult((props) => <div><input type="file" name='ddd'/>InSide</div>);

const result2 = demoResult((props) => <h3>Config</h3>);

const result3 = demoResult((props) => <h3>{JSON.stringify(props)}</h3>);

export default [
  {
    label: "Ejemplo",
    render() {
      return (
        <host>
          <a-router-proxy>
            <p>
              The import effect is emulated with a delay to show the use of the
              loading slot
            </p>
            <nav>
              <a style="margin:5px" href="/">
                home
              </a>
              <a style="margin:5px" href="/user">
                user
              </a>
              <a style="margin:5px" href="/config">
                config
              </a>
            </nav>
            <a-router-switch>
              <a-router-case path="/" src={result1}></a-router-case>
              <a-router-case path="/user" src={result2}></a-router-case>
              <a-router-case path="/:any..." src={result3}></a-router-case>
              <h1 slot="loading">loading...</h1>
              <h1 slot="404">404</h1>
            </a-router-switch>
          </a-router-proxy>
          <div><input type="file" name='outsize'/>Outsize</div>
        </host>
      );
    },
  },
];

```
https://github.com/ranglang/atomico-kit/commit/02cc9c0f7c98d9ebc28dbc2b5a83627fd97eeda7

![7rhar-416wh](https://user-images.githubusercontent.com/12862508/82268171-edb37680-99a0-11ea-8fd6-d6f67837ed0f.gif)

**Comment by UpperCod** [comment]
Hi, I apologize for not responding earlier. thanks for detecting this behavior, I will look for a solution to this bug within the day, Thank you very much for using Atomico!.

**Comment by UpperCod** [comment]
Problem solved in version 0.3.2, it is not a bug of the Atomico core, it was a mishandling of the onclick event associated with the component `a-router-proxy`.

`a-router-proxy` captures click events to redirect the target path, if it declares the href attribute

Thanks for your issue

**Comment by ranglang** [comment]
thanks you
